### ISSUE TITLE
Build error using htm library with Preact

### ISSUE THREAD

**Issue by chimon2000** [egoist/poi#552]
I get the following error when running `poi dev` after setting up a Preact app with the [htm](https://github.com/developit/htm) library and typescript.

![image](https://user-images.githubusercontent.com/6907797/55502781-59127b80-561b-11e9-9e7d-8d68624605f6.png)

For an example of the code that I have written.

```
import './assets/css/style.css'
import { html, render } from 'htm/preact'
import { App } from './App'

const app = document.getElementById('app') || document.body

render(
    html`
        <${App} />
    `,
    app
)
```

**Comment by egoist** [comment]
does it work without typeacript?

**Comment by chimon2000** [comment]
I get the same error without TypeScript.  Here is my full `index.js`

```
import { html, render, Component } from 'htm/preact'

export class App extends Component {
    render() {
        console.log('here')
        return html`
            <div>
                <h1 class="title">Hello Poi!</h1>
                <div class="tip">Edit src/index.ts and save to reload.</div>
            </div>
        `
    }
}

const app = document.getElementById('app') || document.body
render(
    html`
        <${App} />
    `,
    app
)

```

**Comment by egoist** [comment]
Oh I think you need this webpack plugin: https://github.com/lukeed/webpack-modules

**Comment by egoist** [comment]
Fixed in 12.6.1, you don't need that plugin now.
